cpp/tests/replace/replace_tests.cpp

```diff
@@ -47,8 +47,11 @@ TEST_F(ReplaceErrorTest, SizeMismatch)
   cudf::test::fixed_width_column_wrapper<int32_t> values_to_replace_column{{10, 11, 12, 13}};
   cudf::test::fixed_width_column_wrapper<int32_t> replacement_values_column{{15, 16, 17}};
 
-  EXPECT_THROW(cudf::find_and_replace_all(
-                 input_column, values_to_replace_column, replacement_values_column, mr()),
+  EXPECT_THROW(cudf::find_and_replace_all(input_column,
+                                          values_to_replace_column,
+                                          replacement_values_column,
+                                          cudf::test::get_default_stream(),
+                                          mr()),
```

Comment:
Why was mr being passed here? Maybe this should be removed?

Reply:
Good point, I assumed this was a meaningful mr but it's just pulling [the mr set on the base fixture](https://github.com/rapidsai/cudf/blob/branch-23.10/cpp/include/cudf_test/base_fixture.hpp#L59). Do you know why that mr even exists? It's used pretty sparingly, and given its definition it seems like we could probably just remove it altogether. What do you think?

Reply:
I think we can remove it. I'm not seeing a purpose for it here.

Reply:
I'll update these files in this PR and make a follow-up to remove from the fixture.

Reply:
Done.